assets/javascripts/discourse/models/landing-page.js.es6

```diff
@@ -58,7 +69,7 @@ LandingPage.reopenClass({
   },
 
   import(data) {
-    return ajax(`${basePath}/import`, {
+    return ajax(`${basePath}/upload`, {
```

Comment:
@cabidop Is there a need to change this path?

Reply:
It seems so, the [routes.rb](https://github.com/paviliondev/discourse-landing-pages/blob/main/config/routes.rb#L8) file only defines the `POST :page/upload` path, which also matches the `upload` method in the [`PageController`](https://github.com/paviliondev/discourse-landing-pages/blob/main/app/controllers/landing_pages/admin/page.rb#L55).

Reply:
Ok.